components/molecules/imageLocation/__tests__/stories.js

```diff
@@ -9,14 +9,15 @@ import React from 'react'
 import { object, select, withKnobs } from '@storybook/addon-knobs'
 import { action } from '@storybook/addon-actions'
 import { Wrapper } from 'decorators'
-
 // UI
 import { ImageLocation } from 'components'
 import Readme from '../README.md'
 
 // Data
 import { Item } from '../__mocks__/itemFloor'
-
+import { useForm } from 'react-hook-form'
+import { Button, Alert } from 'index'
+import { ImageLocationFormElement } from '../components/imageLocationFormElement'
```

Comment:
@mhsnprvr2 Can just use from 'components' in stories, unless this is only for this story?

> import { ImageLocation } from 'components'